client/src/modules/Schedule/components/FilteredTags/index.tsx

```diff
@@ -0,0 +1,3 @@
+import { FilteredTags } from './FilteredTags';
+
+export default FilteredTags;
```

Comment:
Perhaps, we could just use construction `export ... from ...` here in single line